server/game/CardAbility.js

```diff
@@ -9,16 +9,17 @@ class CardAbility extends ThenAbility {
         if(properties.initiateDuel) {
             if(properties.condition) {
                 let condition = properties.condition;
-                properties.condition = context => context.source.isParticipating() && condition(context);
+                properties.condition = context => (context.source.type !== CardTypes.Character || context.source.isParticipating()) && condition(context);
```

Comment:
Given the target conditions, do we need these extra condition checks? (I know that's not something you added, just pointing it out)

Reply:
Yeah I think you are right.  I'll push a new commit.